### ISSUE TITLE
[Imprv] add functionality to set network mac_address to module ntnx_vms

### ISSUE THREAD

**Issue by kenmoini** [nutanix/nutanix.ansible#201]
**Describe the request**

Assigning MAC addresses to network interfaces is crucial for some workloads such as reserved DHCP requests and day 1 static networking in OpenShift with the Assisted Installer Service.

**Current behaviour**

Currently, the ntnx_vms module does not allow setting the MAC address property, only a private IP if it is a managed subnet.

**Expected behaviour**

Expected behavior should extend the subnet definition with a MAC Address to allow enterprise IPAM network mapping.

```yaml
  - name: create Vm
    nutanix.ncp.ntnx_vms:
      state: present
      name: "ansible_automation_demo"
      networks:
        - is_connected: True
          mac_address: "50:6B:8D:69:42:04"
          subnet:
            name: "{{ subnet_name }}"
```

**Comment by kenmoini** [comment]
For reference, I have the functionality employed in my fork in a separate branch: https://github.com/kenmoini/nutanix.ansible/tree/kenmoini-mac-address-patch

The code is there, it works, just need guidance on what remains to contribute via PR - I am unaware of what the Nutanix-side integration testing would require.

**Comment by bhati-pradeep** [comment]
Changes released in v1.7.0
